src/components/ContactDetails/ContactDetails.tsx

```diff
@@ -1,13 +1,126 @@
 import React from 'react';
 import { useParams } from 'react-router-dom';
+import {
+  makeStyles,
+  TextareaAutosize,
+  Typography,
+  CircularProgress,
+  Box,
+} from '@material-ui/core';
+import gql from 'graphql-tag';
+import { useQuery } from '@apollo/react-hooks';
+import NavBar from '../NavBar';
+import Avatar from '../Avatar';
+import { ReactComponent as EllipseShape } from '../../svg/ellipse-shape-gray.svg';
+import { ROUTES } from '../../shared/routes';
+import { ContactDetailsParams, ContactDetailsProps, NewbieData } from './types';
 
-const ContactDetails: React.FC = () => {
-  const { id: newbieId } = useParams();
+const useStyles = makeStyles(theme => ({
+  bold: {
+    fontWeight: 700,
```

Comment:
We can follow the design tokens defined in the teme to pick different font weights.
For example, taking a look at https://material-ui.com/system/typography/, we can verify that we have available following fontWeights:

- fontWeightLight: 300
- fontWeightRegular: 400
- fontWeightMedium: 500
- fontWeightBold: 700